src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystemManagerImplementationFactory.py

```diff
@@ -1,5 +1,5 @@
 #
-#   Copyright 2013-2021 The Foundry Visionmongers Ltd
+#   Copyright 2013-2020 The Foundry Visionmongers Ltd
```

Comment:
[Fix] Assuming this is a typo, should be 2022?

Reply:
Lol yes, that was `<Ctrl-A>` instead of `<Ctrl-X>`... great spot. Fixed [here]()https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/72c05c583b2cb37d4a06d3c8aab9fa1dd96aa9c1).